app/util/test/testSetup.js

```diff
@@ -269,6 +270,11 @@ jest.mock('react-redux', () => ({
   ...jest.requireActual('react-redux'),
 }));
 
+jest.mock('react-native-default-preference', () => ({
+  get: jest.fn(),
+  set: jest.fn(),
+}));
+
 // eslint-disable-next-line import/no-commonjs
 /* require('react-native-reanimated/lib/reanimated2/jestUtils').setUpTests(); */
 /* global.__reanimatedWorkletInit = jest.fn(); */
```

Comment:
Not related to your changes but we can replace these two lines with
```suggestion
require('react-native-reanimated/lib/module/reanimated2/jestUtils').setUpTests();
global.__reanimatedWorkletInit = jest.fn();
```